src/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+.snake {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block__text {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 300px;
+  height: 300px;
+  text-align: center;
+  font-size: 100px;
+  color: white;
+}
+
+.block__text:nth-child(1) {
+  background-color: rgba(255,0,0,255);
+}
+
+.block__text:nth-child(2) {
+  background-color: rgba(204,0,0,255);
+}
+
+.block__text:nth-child(3) {
+  background-color: rgba(153,0,0,255);
+}
+
+.block__text:nth-child(4) {
+  background-color: rgba(102,0,0,255);
+}
+
+.block__text:nth-child(5) {
+  background-color: rgba(51,0,0,255);
+}
+
+.block__text:nth-child(6) {
+  background-color: rgba(0,0,0,255);
+}
+
+@media (min-width: 900px) {
+  .snake {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .block__text {
+    flex: 33%;
+  }
+
+  .block__text:nth-child(4) {
+    order: 5;
+  }
+
+  .block__text:nth-child(5) {
+    order: 4;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
+  .snake {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .block__text {
+    flex: 50%;
+  }
+
+  .block__text:nth-child(1) {
+    order: 1;
+  }
+
+  .block__text:nth-child(2) {
+    order: 2;
+  }
+
+  .block__text:nth-child(3) {
+    order: 4;
+  }
+
+  .block__text:nth-child(4) {
+    order: 3;
+  }
+
+  .block__text:nth-child(5) {
+    order: 5;
+  }
+
+  .block__text:nth-child(6) {
+    order: 6;
+  }
+}
+
+@media (max-width: 599px) {
+  .snake {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .block__text {
+    width: 100vw;
+  }
```

Comment:
you can also use flex basis. it makes more sense if you are using a flex container